src/main/java/dataLoader/csvDataLoader.java

```diff
@@ -0,0 +1,33 @@
+package dataLoader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+/**
+ * Simple class that parses data set from a csv file
+ * without knowing the headers of the file
+ * */
+
+public class csvDataLoader {
+
+    private String location = "/path/to/file.csv";
+    private Reader csvData = new StringReader(location);
+
+    public void parseCsvData( String csv) throws IOException {
```

Comment:
CSV stands for Comma Serated Values. Nonetheless, it is quite frequent that people also use spaces, tabs or semicolons to distinguish columns. How do we handle this?

Reply:
Ok we parsed the data but where do we store for the application to access it?

Reply:
check this [out](https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.html#DEFAULT). I think it'll do the handling for us.